sig-and-wg/sig-core/proposals/event-bus-monitoring-metrics.md

```diff
@@ -0,0 +1,113 @@
+# Event Bus Monitoring Metrics Proposal
+
```

Comment:
Please add a description of what problem we are trying to solve here?
e.g. `The metrics aim to provide support with troubleshooting when there are issues with event delivery. As a developer, I should be able to look at the metrics and figure out why events are not delivered for my lambda/serverless.`

Reply:
I don't think the metrics docs should look like that because of:
- Metrics documentation usually is a plain description of what each metric describe, not the scenarios it might help with like https://docs.datadoghq.com/integrations/kafka/
- The troubleshooting scenarios can be ina different document format, each scenario and how to tackle and what metrics to look into
- Troubleshooting some scenarios would require looking into more that a metric chart and deduce what's wrong.
- Anomaly in any metric can be indicator for a problem.  
I'd suggest to have the metrics doc as a key-value (name-desc) table and move the troubelshooting scenarios and how the metrics can help to the troubleshooting guide.

Reply:
This is a proposal. It is better to provide a context and description. And here the intention is to discover those metrics that can support troubleshooting. Just providing a plain list is not what will help.

Reply:
Added the motivation per metric

Reply:
Done